crates/tabby/src/model.rs

```diff
@@ -0,0 +1,55 @@
+use clap::Subcommand;
+use tabby_common::registry::{parse_model_id, ModelRegistry, DEFAULT_MODEL_REGISTRY_NAME};
+
+#[derive(Subcommand)]
+pub enum ModelArgs {
+    #[clap(alias = "rm")]
```

Comment:
Overall looks good, but could you improve the commandline interface by adding `///` style comments to each sub model args?